pkg/api/webassets/webassets.go

```diff
@@ -29,26 +32,64 @@ type EntryPointInfo struct {
 
 var entryPointAssetsCache *dtos.EntryPointAssets = nil
 
-func GetWebAssets(cfg *setting.Cfg) (*dtos.EntryPointAssets, error) {
+func GetWebAssets(cfg *setting.Cfg, license licensing.Licensing) (*dtos.EntryPointAssets, error) {
 	if cfg.Env != setting.Dev && entryPointAssetsCache != nil {
 		return entryPointAssetsCache, nil
 	}
 
-	result, err := readWebAssets(filepath.Join(cfg.StaticRootPath, "build", "assets-manifest.json"))
-	entryPointAssetsCache = result
+	var err error
+	var result *dtos.EntryPointAssets
+
+	cdn := "" // "https://grafana-assets.grafana.net/grafana/10.3.0-64123/"
```

Comment:
Original comment by Ryan that I deleted by accident; this is for testing only for the time being